voice/bxml/verbs/playAudio.md

```diff
@@ -21,7 +21,7 @@ header is set on the response, media will not be cached.
 ### Text Content
 | Name     | Description                                                                                                                                  |
 |:---------|:---------------------------------------------------------------------------------------------------------------------------------------------|
-| audioUri | The URL of the audio file realative to the endpoint that returned the BXML <br> ⚠️ **ONLY** .wav files encoded as PCM or G711 are supported. |
+| audioUri | The URL of the audio file relative to the endpoint that returned the BXML <br> ⚠️ **ONLY** .wav files encoded as PCM or G711 are supported. |
```

Comment:
This is confusing. Can we make the wording more similar to the wording for callbacks? Like "The URL of the audio file. May be a relative URL." or similar?